docs/tables/gcp_dns_record_set.md

```diff
@@ -0,0 +1,35 @@
+# Table: gcp_dns_record_set
+
+A record set (also known as a resource record set) is the collection of DNS records in a zone that have the same name and are of the same type. Most record sets contain a single record.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name, 
+  managed_zone_name, 
+  type, 
+  kind, 
+  routing_policy,
+  rrdatas,
+  signature_rrdatas,
+  ttl
+from
+  gcp_dns_record_set;
+```
+
+### List record sets of type 'A'
```

Comment:
We should use some Type which are important from [here](https://cloud.google.com/dns/docs/records#record_type) 
such as CNAME or DNSKEY

Reply:
Updated the type filter with CNAME